R/functions.R

```diff
@@ -188,7 +188,7 @@ summarizeAtlasExperiment <- function( experimentAccession, atlasExperimentDirect
 	atlasExperimentSummary <- lapply( allAnalytics, function( analytics ) {
 		
 		# Get the SDRF rows for this analytics object's assays.
-		analyticsSDRF <- .createAnalyticsSDRF( analytics, atlasSDRF )
```

Comment:
Ah, your making the hidden `.` prefix function to normal one to export this function